apps/whale-api/src/module.api/poolswap.pathfinding.service.ts

```diff
@@ -58,20 +60,27 @@ export class PoolSwapPathFindingService {
           toToken: toToken,
           bestPath: path,
           estimatedReturn: formatNumber(
-            computeReturnInDestinationToken(path, fromTokenId)
+            computeReturnLessDexFeesInDestinationToken(path, fromTokenId).estimatedReturn
+          ), // denoted in toToken
+          estimatedReturnLessDexFees: formatNumber(
+            computeReturnLessDexFeesInDestinationToken(path, fromTokenId).estimatedReturnLessDexFees
```

Comment:
`computeReturnLessDexFeesInDestinationToken` can just be called once here instead of twice